src/applications/vaos/tests/e2e/page-objects/ClinicChoicePageObject.js

```diff
@@ -6,10 +6,15 @@ class ClinicChoicePageObject extends PageObject {
     return this;
   }
 
-  assertUrl() {
+  assertUrl({ isCovid = false } = {}) {
     cy.url().should('include', '/clinic', { timeout: 5000 });
     cy.axeCheckBestPractice();
 
+    this.clickNextButton();
```

Comment:
Not expecting assertUrl to click a button. I would move the assertion to the test.